### ISSUE TITLE
App flashes and disappears after login

### ISSUE THREAD

**Issue by timo-tech19** [adrianhajdin/project_shareme_social_media#8]
After I signed in, the app goes to the home page but then the content disappears after a few seconds. On checking the dom tree in the dev tools, the root div is emptied after the app loads.

**Comment by Vidhu007** [comment]
Yes, I am facing the same issue! White screen appears after successfully logging

**Comment by thatmissedsemicolon** [comment]
its because your length is undefined. adding a {save?.length} will fix your issue. Hope it helps!

**Comment by nasyx-rakeeb** [comment]
where to add it

**Comment by thatmissedsemicolon** [comment]
> it

Check your console what exact error are you getting.

**Comment by nasyx-rakeeb** [comment]
I am getting 2 errors i took a Screenshot of console check it out

![Screenshot_20220529-084153_VNC Viewer](https://user-images.githubusercontent.com/89723867/170850377-3d4aeaa3-c335-4aba-97fd-255c1f28b5f6.jpg)

**Comment by thatmissedsemicolon** [comment]
What exactly your question is? Did you try to attempt yourself and it's not published by me.> I am getting 2 errors i took a Screenshot of console check it out and 1 more thing, its not happening on local environment its the same app which u have published this problem is happening on your production version
> 
> 
> 
> ![Screenshot_20220529-084153_VNC Viewer](https://user-images.githubusercontent.com/89723867/170850377-3d4aeaa3-c335-4aba-97fd-255c1f28b5f6.jpg)
> 
>

**Comment by nasyx-rakeeb** [comment]
what i am trying to say is i haven't cloned this repo this problem is happening on main project i mean on this url:
[https://share-me-sanity.netlify.app/](https://share-me-sanity.netlify.app/) 
this url is the original production version of this project

**Comment by nasyx-rakeeb** [comment]
try to open this url and see the problem

**Comment by thatmissedsemicolon** [comment]
The error you should be able to see on console it's says property null that means something on the code is undefined that's why you are not able to see it. Specially something in image URL is not defined when importing from sanity's client. I would recommend give yourself a try not every-time it will work since a lot changes happens over time. If you try to code yourself and figure out through console you should be able to successfully see everything without errors.

**Comment by nasyx-rakeeb** [comment]
Thanks i will try

**Comment by amsalimsema** [comment]
@nasyx-rakeeb Did you get passed that error? Thanks

**Comment by nasyx-rakeeb** [comment]
@amsalimsema no, nothing seems to work
